src/test/java/com/exactpro/junit5/WithByteBufInvocationContext.java

```diff
@@ -0,0 +1,61 @@
+package com.exactpro.junit5;
+
+import io.netty.buffer.ByteBuf;
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.TestTemplateInvocationContext;
+
+import java.util.Collection;
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+
+public class WithByteBufInvocationContext implements TestTemplateInvocationContext {
+
+    private final List<ByteBuf> arguments;
+
+    private StringBuilder displayName;
+
+    private WithByteBufMethodContext methodContext;
+
+    WithByteBufInvocationContext(String methodName, WithByteBufMethodContext methodContext, List<ByteBuf> arguments) {
+        this.displayName = new StringBuilder(methodName);
+        this.arguments = arguments;
+        this.methodContext = methodContext;
+    }
+
+    @Override
+    public String getDisplayName(int invocationIndex) {
+        return formatName(arguments);
+    }
+
+    @Override
+    public List<Extension> getAdditionalExtensions() {
+        return singletonList(new WithByteBufParameterResolver(methodContext, this.arguments));
+    }
+
+    private String formatName(Collection<ByteBuf> arguments) {
+        for (ByteBuf buf : arguments) {
+            appendFormattedBuffer(buf);
+        }
+        return displayName.toString();
+    }
+
+    private void appendFormattedBuffer(ByteBuf buf) {
+        int readerIndex = buf.readerIndex();
+        int readLimit = buf.writerIndex();
+        displayName.append(" [ ");
+        while (readerIndex < readLimit) {
+            appendHexByte(buf.getByte(readerIndex++));
+        }
+        displayName.append("]");
+    }
+
+    private void appendHexByte(int oneByte) {
+        if (oneByte == 0) {
```

Comment:
It's not only about 0. All 1-digit numbers should be extended to two digits. Add unit tests for formatter